package.json

```diff
@@ -43,7 +43,7 @@
     "lint": "yarn eslint --ext .ts .",
     "lint:fix": "yarn lint -- --fix",
     "lint:fix-cached": "yarn lint:fix -- --cache",
-    "check:license": "yarn license-checker -direct --excludePackages \"@sap/edm-converters@1.0.21;@sap/xsenv@2.2.0;weak-map@1.0.5\" --onlyAllow \"MIT;Apache;ISC;BSD;WTFPL;Artistic;CC-BY;CC0;Unlicense;Public Domain;GPL-3.0\"",
+    "check:license": "node scripts/check-licenses.js",
```

Comment:
btw, you said, this was a fix, so calling the same command with yarn was not possible?

Reply:
It is possible, but because there are workspaces now and license-checker does not support this correctly, local packages are recognized, but because they are private license-checker marks them as "unlicensed". I considered multiple solutions, like other packages or `yarn licenses list` (which might be better in the end, but because it is in yml format I decided against it).

Reply:
Btw I just saw, that yarn has a `--json` flag. I'll check whether I can work with that.

Reply:
I checked and I think it is not worth it.